pkg/acceptance/helpers/row_access_policy_client.go

```diff
@@ -56,7 +55,7 @@ func (c *RowAccessPolicyClient) DropRowAccessPolicyFunc(t *testing.T, id sdk.Sch
 
 // GetRowAccessPolicyFor is based on https://docs.snowflake.com/en/user-guide/security-row-intro#obtain-database-objects-with-a-row-access-policy.
 // TODO: extract getting row access policies as resource (like getting tag in system functions)
-func (c *RowAccessPolicyClient) GetRowAccessPolicyFor(t *testing.T, id sdk.SchemaObjectIdentifier, objectType sdk.ObjectType) (*PolicyReference, error) {
+func (c *RowAccessPolicyClient) GetOneRowAccessPolicyFor(t *testing.T, id sdk.SchemaObjectIdentifier, objectType sdk.ObjectType) (*PolicyReference, error) {
```

Comment:
Should we also move this method? It's not row access policy specific and with `ReferencesClient` extracted it would be better.

Also, why we add `One` and `All` instead of just using singular and plural (Get...Policy vs Get...Policies)?

Reply:
Moved. Changed names to `GetPolicyReference` and `GetPolicyReferences`.